cmd/controller-manager/app/client_builder.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package controller
+package app
```

Comment:
Now that https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/controller-manager exists, can we move it there rather than under cmd/controller-manager?